includes/widgets/playing-cards.php

```diff
@@ -0,0 +1,500 @@
+<?php
+
+namespace Elementor;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+/**
+ * Elementor playing card`s widget.
+ *
+ * Elementor widget playing cards.
+ *
+ * @since 1.0.0
+ */
+class Widget_Playing_Cards extends Widget_Base {
+
+
+	/**
+	 * Get widget name.
+	 *
+	 * Retrieve sidebar widget name.
+	 *
+	 * @return string Widget name.
+	 * @since 1.0.0
+	 * @access public
+	 *
+	 */
+	public function get_name(): string {
+		return 'playing-cards';
+	}
+
+	/**
+	 * Get widget title.
+	 *
+	 * Retrieve sidebar widget title.
+	 *
+	 * @return string Widget title.
+	 * @since 1.0.0
+	 * @access public
+	 *
+	 */
+	public function get_title(): string {
+		return esc_html__( 'Playing Cards', 'elementor' );
+	}
+
+	/**
+	 * Get widget icon.
+	 *
+	 * Retrieve playing cards widget icon.
+	 *
+	 * @return string Widget icon.
+	 * @since 3.12.1
+	 * @access public
+	 *
+	 */
+	public function get_icon(): string {
+		return 'eicon-heart';
+	}
+
+	/**
+	 * Get widget keywords.
+	 *
+	 * Retrieve the list of keywords the widget belongs to.
+	 *
+	 * @return array Widget keywords.
+	 * @since 2.1.0
+	 * @access public
+	 *
+	 */
+	public function get_keywords(): array {
+		return [ 'card', 'playing cards' ];
+	}
+
+	/**
+	 * Register playing-cards widget controls.
+	 *
+	 * Adds card`s and card styles.
+	 *
+	 * @since 1.0.0
+	 * @access protected
+	 */
+	protected function register_controls(): void {
+		$this->start_controls_section(
+			'cards_section',
+			[
+				'label' => __( 'Cards', 'elementor' ),
+				'tab' => \Elementor\Controls_Manager::TAB_CONTENT,
```

Comment:
We generally add a `use Elementor\Controls_Manager;` line after the namespace, and then just write `Controls_Manager::TAB_CONTENT` to save some line space

Reply:
You don't need to import in Core widget. It's already in the correct namespace.

Reply:
fixed